web/src/locales/en.json

```
@@ -1356,8 +1356,8 @@
       "zoneName": "Montenegro"
     },
     "MF": {
-      "countryName": "Saint Martin",
-      "zoneName": "French"
+      "countryName": "France",
+      "zoneName": "Saint-Martin"
```

Since this has it's own ISO code it should just be:
```suggestion
      "zoneName": "Saint-Martin"
```

Note however that this zone is currently not added to Electricity Maps, due to it's size and lack of data.